File: Java Food Order App/Restaurant.java
package FoodOrderApp;

import java.time.LocalDate;
import java.time.LocalTime;
import java.time.format.DateTimeFormatter;

public class Restaurant {
	String gstNo;
	String restaurantName;
	Address restaurantAddress;
	long phoneNo;
	
	Restaurant(String gst, String name, Address add, long phone) {
		gstNo = gst;
		restaurantName = name;
		restaurantAddress = add;
		phoneNo = phone;
	}
	
	void display() {
		System.out.println("\n------- Welcome to " + this.restaurantName + " --------");
		System.out.println("Address: " + this.restaurantAddress);
		System.out.println("\nPhone: " + this.phoneNo + "      GST No: " + this.gstNo);
		System.out.println("\t\tBill No: 11970  ");
		System.out.print("Date: " + LocalDate.now().format(DateTimeFormatter.ofPattern("dd/MM/yyyy")));
		System.out.println("\tTime: " + LocalTime.now().toString().substring(0, 8));
	}
}

File: Kotlin Food Order App/Restaurant.kt
package foodOrderApp

import java.time.LocalDate
import java.time.LocalTime
import java.time.format.DateTimeFormatter

class Restaurant(gst: String,name: String, address: String, phone: Long ) {
    private var restaurantName: String = ""
    private var gstNo: String = ""
    private var address: String = ""
    private var phoneNo: Long = 0L

    init {
        this.restaurantName = name
        this.gstNo = gst
        this.address = address
        this.phoneNo = phone
    }

    fun display() {
        println("\n------- Welcome to " + this.restaurantName + " --------");
        println("Address: ${this.address}");
        println("\nPhone: ${this.phoneNo}     GST No: ${this.gstNo}");
        println("\t\tBill No: 11970  ");
        print("Date: " + LocalDate.now().format(DateTimeFormatter.ofPattern("dd/MM/yyyy")));
        println("\tTime: " + LocalTime.now().toString().substring(0, 8));
    }
}